sdk/ts/orm/prisma.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: Prisma
+sidebarTitle: Prisma
+description: Configure Prisma to work with your Turso database
+---
+
+![Prisma banner](/images/guides/prisma-banner.png)
+
+## Prerequisites
+
+Before you start, make sure you:
+
+- [Install the Turso CLI](/cli/installation)
+- [Sign up or login to Turso](/cli/authentication#signup)
+- Prisma versions 5.4.2 and later
+
+<Steps>
+
+<Step title="Install the libSQL SDK and its Prisma driver">
+
+<CodeGroup>
+
+```bash npm
+npm install @libsql/client @prisma/adapter-libsql
+```
+
+```bash pnpm
+pnpm add @libsql/client @prisma/adapter-libsql
+```
+
+```bash yarn
+yarn add @libsql/client @prisma/adapter-libsql
+```
+
+</CodeGroup>
+
+</Step>
+
+<Step title="Configure database credentials">
+
+Get the database URL:
+
+```bash
+turso db show --url <database-name>
+```
+
+Get the database authentication token:
+
+```bash
+turso db tokens create <database-name>
+```
+
+Assign credentials to the environment variables inside `.env.local`:
+
+```bash
+TURSO_DB_URL="..."
```

Comment:
These don't match what is used in the `createClient` below. The ones in `createClient` though are the format of the new names we should be using:

* `TURSO_DATABASE_URL`
* `TURSO_AUTH_TOKEN `